CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Version 2.1.1 - October 1, 2018
+=================================
+- Reverted Android firebase-core dependency back to 16.0.1
```

Comment:
Can you add a description as to why? It might help devs figure out if they should upgrade to the patch.